client/src/components/PageComponents/Header/Header.tsx

```diff
@@ -8,21 +9,67 @@ import {
 } from "./Header.css";
 
 const Header = () => {
+  const navigate = useNavigate();
+
+  const gotoMainPage = () => {
+    navigate("/");
+  };
+
+  const gotoSchedulePage = () => {
+    navigate("/schedule");
+  };
+
+  const gotoCalendarPage = () => {
+    navigate("/calendar");
+  };
+
+  const gotoTodolistPage = () => {
+    navigate("/todolist");
+  };
+
+  /* 모달들은 스타일을 확인할 수 있게 새 페이지로 연결만 시켜놓았습니다. */
+  const openLoginModal = () => {
+    navigate("/loginmodal");
+  };
+
+  const openUserEditModal = () => {
+    navigate("/usereditmodal");
+  };
+  /* ............................................................... */
+
   return (
     <div className={headerFooterContainer}>
-      <div className={mainLogo}>SMART DAY</div>
+      <div className={mainLogo} onClick={gotoMainPage}>
+        SMART DAY
+      </div>
       <div className={headerNavigator}>
-        <div className={headerNaviButtons}>전체 일정</div>
+        <div className={headerNaviButtons} onClick={gotoSchedulePage}>
+          전체 일정
+        </div>
         <div>|</div>
         <div className={headerNaviButtons}>날씨</div>
         <div>|</div>
-        <div className={headerNaviButtons}>캘린더</div>
+        <div className={headerNaviButtons} onClick={gotoCalendarPage}>
+          캘린더
+        </div>
         <div>|</div>
-        <div className={headerNaviButtons}>TodoList</div>
+        <div className={headerNaviButtons} onClick={gotoTodolistPage}>
+          TodoList
+        </div>
       </div>
       <div className={headerTwoButtons}>
-        <button className={headerButton}>로그인</button>
-        <button className={headerButton}>회원가입</button>
+        <button
+          className={headerButton}
+          onClick={openLoginModal} //
+        >
+          로그인
+        </button>
```

Comment:
버튼에는 type 속성을 사용해주시는 습관을 들이시면 좋습니다~
```suggestion
        <button
          type='button'
          className={headerButton}
          onClick={openLoginModal} //
        >
          로그인
        </button>
```